### ISSUE TITLE
ORDER BY results in empty results, cmd query tool

### ISSUE THREAD

**Issue by solutionroute** [genjidb/genji#438]
### What version of Genji are you using?

<pre>
$ genji version
Genji v0.14.0
Genji CLI v0.14.0
</pre>

Same behaviour with Boltdb/Badgerdb. To be clear, this is only in the genji cmd tool; I'm not seeing issues using the Genji API.

### What did you do?

Using the command line query tool:

Example data:
> select * from person limit 1;
{
  "sid": "af879a5vfpdad6wd",
  "firstname": "Bolt0",
  "lastname": "Smith",
  "primaryemail": "foo-0@example.com"
}

Example query without ORDER BY
genji> select sid, firstname from person  limit 5;
{
  "sid": "af879a5vfpdad6wd",
  "firstname": "Bolt0"
}
{
  "sid": "af879a5vfpdad6we",
  "firstname": "Bolt1"
}
{
  "sid": "af879a5vftdad6wf",
  "firstname": "Bolt2"
}

### What did you expect to see?

5 documents, ordered ASC or DESC

### What did you see instead?

FAILing query - it doesn't matter what field or direction the order by is issued, always empty N results.
genji> select sid, firstname from person ORDER BY  sid limit 5;
{}
{}
{}
{}
{}

**Comment by asdine** [comment]
Thanks for reporting, I managed to reproduce it. I'll investigate and release a fix soon

**Comment by asdine** [comment]
This has been fixed by v0.14.1, you can run:

```bash
go get github.com/genjidb/genji@latest
go install github.com/genjidb/genji/cmd/genji@latest
```
